### ISSUE TITLE
Final tuning of "enonic project create"

### ISSUE THREAD

**Issue by sigdestad** [enonic/enonic-cli#103]
Below is a suggestion for the order of texts, and the flow. Custom texts are written with quotes.
Most of the old texts are removed.
-----
?initial questions

"Initializing project..." (new text)
Enumerating..
Counting...
Compressing..
Total...
"Project created in '/Users/tsi/myproject'" (new text)

?sandbox questions...

"Initializing sandbox..." (new text)
Downloading Enonic XP.... (adjusted text)
Unzipping...Done
"Sandbox 'mysandbox' created" (new text)
"Project linked to 'mysandbox'"  (new text)
$

-----
So, short version is, after asking questions:
- We inform the user we start doing something
- We inform the user what is going on
- We inform the user it was completed

If only one thing happens, we only inform it was completed.
We should be able to reuse this strategy for all wizards.
